src/t_set.c

```diff
@@ -1420,6 +1420,36 @@ void sinterCommand(client *c) {
     sinterGenericCommand(c, c->argv+1,  c->argc-1, NULL, 0, 0);
 }
 
+/* SMEMBERS key */
+void smembersCommand(client *c) {
+    setTypeIterator *si;
+    char *str;
+    size_t len;
+    int64_t intobj;
+    int encoding;
+    robj *setobj = lookupKeyRead(c->db, c->argv[1]);
+    if (checkType(c,setobj,OBJ_SET)) return;
+    if (!setobj) {
+        addReply(c, shared.emptyset[c->resp]);
+        return;
+    }
+
+    const unsigned long scard = setTypeSize(setobj);
+
+    // Prepare the response
+    addReplyArrayLen(c,scard);
```

Comment:
```suggestion
```suggestion
    /* Prepare the response. */
    addReplySetLen(c,setTypeSize(setobj););
```
```

Reply:
added it but fixing the typo on `suggestion` and `etTypeSize(setobj););`

Reply:
😅mistake.